### ISSUE TITLE
Access more Google Place data

### ISSUE THREAD

**Issue by howiec721** [oscarmcm/django-places#30]
### DESCRIPTION
Is it possible to get access to the JSON from Google? This is a great library but would love to be able to store the address and opening times for business, etc.

**Comment by oscarmcm** [comment]
Currently not out of the box, but I can add support for this. Question: do you want this data to be show in the user input?

**Comment by andrewgy8** [comment]
This is an interesting feature. I would too like to be able to grab the opening times data. Any idea what it would take to do this? Im thinking if there is some way to hook into the api call and once done, you can do whatever you want with the data.

**Comment by oscarmcm** [comment]
@andrewgy8 This might require to move away from a `CharField` to a `JSONField` in order to store more information from the google maps API and maybe a new Setting that stores which data you want from the API.

**Comment by andrewgy8** [comment]
Yeah, that could work. You are thinking of POSTing all the returned data to a field on the model and then the user can do whatever he/she wants with it on the server side?

**Comment by daweedm** [comment]
Hello, any news about this feature ?
